linfa-clustering/src/k_means/hyperparameters.rs

```diff
@@ -95,13 +112,19 @@ impl<F: Float, R: Rng + Clone> KMeansHyperParams<F, R> {
 
     pub fn new_with_rng(n_clusters: usize, rng: R) -> KMeansHyperParamsBuilder<F, R> {
         KMeansHyperParamsBuilder {
+            n_runs: 10,
```

Comment:
I would use `n_runs: 1` as initialization

Reply:
otherwise the algorithm would be 10 times slower in default settings

Reply:
ah just saw that Sklearn is doing the same thing ..

Reply:
Yeah... I suppose it is a right trade-off drawn from experience.